src/main/java/de/plushnikov/intellij/plugin/processor/modifier/FieldDefaultsModifierProcessor.java

```diff
@@ -51,7 +51,7 @@ public boolean isSupported(@NotNull PsiModifierList modifierList) {
 
   @Override
   public void transformModifiers(@NotNull PsiModifierList modifierList, @NotNull final Set<String> modifiers) {
-    if (modifiers.contains(PsiModifier.STATIC)) {
+    if (UtilityClassModifierProcessor.isModifierListSupported(modifierList) || modifiers.contains(PsiModifier.STATIC)) {
```

Comment:
I would like to flip the comparing order of this checks, because if modifiers contains STATIC already we can skip entire processing

Reply:
True, that's definitely better. Will fix